.gitignore

```diff
@@ -10,3 +10,5 @@ local.properties
 build
 **/*.iml
 target
+dependency-reduced-pom.xml
```

Comment:
This definitely isn't blocking, but I remember this pom is generated from `maven-shade-plugin` when it takes out the shaded libraries, do we need to publish/use this pom when publishing the shaded jar?